src/json/pull_parser.cr

```diff
@@ -1,4 +1,52 @@
 # This class allows you to consume JSON on demand, token by token.
+#
+# Each *read_** method consumes the next token.
```

Comment:
In most markdown parsers this should be `*read_\**` but I'm not sure how the docs generator renders it. Did you preview this?

Reply:
I tried with `*read_\**`, but it is rendered as _read\_\\*_. Surprisingly `*read_**` just works.

Reply:
Okay, it's fine then.